src/panels/lovelace/editor/card-editor/hui-dialog-edit-card.ts

```diff
@@ -335,6 +348,22 @@ export class HuiDialogEditCard
     this._cardEditorEl?.focusYamlEditor();
   }
 
+  private get _isInSection() {
+    return this._params!.path.length === 2;
+  }
```

Comment:
I'm not a fan of this, it means custom views will always be shown in the default section layout. I think instead the preview element could be provided by the view optionally, then that can decide how to render the preview.

Reply:
We use the type of the section parent.